README.md

```diff
@@ -2,15 +2,18 @@
 
 The computing pipeline is a data storage and computing system that provides researchers with access to all of the ‘raw’ data and derived plant phenotypes.
 
-This repository is the central issue repository for the AgPipeline organization.
+This repository is the central issue repository, and technical documentation landing site, for the AgPipeline organization.
```

Comment:
Is the first paragraph (line 3) an accurate description / introduction to the AgPipeline?

Reply:
@dlebauer What do you think of the following for the first paragraph:
"This computing pipeline is a system for processing data to produce derived plant phenotypes."

Reply:
"remote sensing data and derived plant phenotypes".

Reply:
Done